indexer/package.json

```
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/akash-network/cloudmos",
   "dependencies": {
+    "@akashnetwork/akash-api": "file:./akashnetwork-akash-api-0.0.0.tgz",
```

To be updated when `akash-api` is published
```suggestion
    "@akashnetwork/akash-api": "@akashnetwork/akash-api@^1.0.0",
```